wdl/subworkflows/get_preseq_metrics.wdl

```diff
@@ -7,15 +7,16 @@ import "../tasks/preseq.wdl"
 
 workflow get_preseq_metrics {
     input {
+        Boolean preseq_defects_mode = false
         File nuclear_bam
-        PreseqLcExtrapParams params = object {
```

Comment:
Hmm, this is technically here so you can override it when called. I guess we are never going to do that. I wonder if it makes sense to keep it separate from params since it is user defined?

Reply:
You are suggesting we would pull the `defects` out from `PreseqLcExtrapParams`, into its own `Boolean`, right?

Reply:
Yeah, kind of torn because `PreseqLcExtrapParams` is supposed to bundle these together. But this is the one special user defined parameter, we fix the others. If we split it out then we can keep the object definition in the inputs, and it could theoretically be overridden. In that case though you would have to add a new special input to the lc_extrap call. Not sure if there is strong case either way. Maybe whatever works for now.

Reply:
Yeah, I suppose just leave it the way it is now, and if actual need to change the other ones arise we rethink.